### ISSUE TITLE
How do you configure this?

### ISSUE THREAD

**Issue by modemgeek** [sjkp/letsencrypt-azure#6]
I see the option to deploy to Azure but it seems that you must use Azure DNS.  If we have godaddy instead, how do we get this configured?  I'm not finding any documentation.  Thanks.

**Comment by MatthewSteeples** [comment]
If you don't want to use Azure DNS then you may be better off using the site extension: https://github.com/sjkp/letsencrypt-siteextension

**Comment by MatthewSteeples** [comment]
Ah sorry, I see GoDaddy is supported. I'll be quiet now 🤐

**Comment by modemgeek** [comment]
Yes, we currently use the site extension.  However we would like to try this out as it supports wildcard certificates.  It also seemed easier to setup but I couldn't find any documentation on  what I am supposed to do to get it configured or how to use it.
